UI/window-youtube-actions.cpp

```diff
@@ -89,6 +92,54 @@ OBSYoutubeActions::OBSYoutubeActions(QWidget *parent, Auth *auth)
 
 	ui->scheduledTime->setDateTime(QDateTime::currentDateTime());
 
+	auto thumbSelectionHandler = [&]() {
+		if (thumbnailFile.isEmpty()) {
+			QString filePath = OpenFile(
+				this,
+				QTStr("YouTube.Actions.ThumbnailSelectFile"),
+				QStandardPaths::writableLocation(
+					QStandardPaths::PicturesLocation),
+				QString("Images (*.png *.jpg *.jpeg *.gif)"));
+
+			if (!filePath.isEmpty()) {
+				QFileInfo tFile(filePath);
+				if (!tFile.exists())
+					return ShowErrorDialog(
+						this,
+						QTStr("YouTube.Actions.Error.FileMissing"));
+				if (tFile.size() > 2 * 1024 * 1024)
```

Comment:
Same comment as earlier.  To avoid confusion, either use curly braces, or separate with a blank line to visually separate.

Reply:
I converted this to an "else if" with braces.